packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -85,33 +74,34 @@ export const MenuList = compose<MenuListType>({
         return child;
       });
 
-      const content =
-        Platform.OS === 'macos' ? (
-          <Slots.root onMouseLeave={setFocusZoneFocus} onKeyDown={menuList.onListKeyDown}>
-            <Slots.scrollView
-              showsVerticalScrollIndicator={menuContext.hasMaxHeight}
-              showsHorizontalScrollIndicator={menuContext.hasMaxWidth}
+      const shouldHaveScrollView = Platform.OS === 'macos' || menuList.hasMaxHeight || menuList.hasMaxWidth;
```

Comment:
We can revisit later, but it might actually be an improvement on macOS to only have a scrollview if maxWidth//height are defined.. aka we could maybe take out the platform check from here lol

Reply:
I unfortunately can't test this so I can't confirm the behavior of the Menu on macOS. So I would say out of scope, if that's ok